homeassistant/components/cambridge_audio/diagnostics.py

```diff
@@ -8,7 +8,7 @@
 
 from . import CambridgeAudioConfigEntry
 
-TO_REDACT = {CONF_HOST}
+TO_REDACT = {CONF_HOST, "udn", "unit_id"}
```

Comment:
I would not really see either as redactable info. As in, a host in itself isn't sensitive, and udn and unit id are just serial numbers (right?) which is okay I think

Reply:
Yeah none of that is considered sensitive. UDN / Unit IDs are effectively just serial numbers. I'll remove this.